svc-events/events/common.go

```diff
@@ -269,6 +269,14 @@ func validateFields(request *evmodel.RequestBody) (int32, string, []interface{},
 		request.Context = evmodel.Context
 	}
 
+	if request.DeliveryRetryPolicy == "" {
+		request.DeliveryRetryPolicy = evmodel.DeliveryRetryPolicy
+	} else if request.DeliveryRetryPolicy == "TerminateAfterRetries" || request.DeliveryRetryPolicy == "SuspendRetries" || request.DeliveryRetryPolicy == "RetryForeverWithBackoff" {
```

Comment:
can we check only what is allowed, instead of checking what is not allowed in DeliveryRetryPolicy